mws/mws.py

```diff
@@ -250,10 +274,7 @@ def make_request(self, extra_data, method="GET", **kwargs):
         # Amazon's MWS does not allow such a thing.
         extra_data = remove_empty(extra_data)
```

Comment:
I missed this part earlier, but I don't see a point to having two lines to command operations on the same object. We can tuck the "remove_empty" call (or the code itself, since it's a one-liner) into the "clean_extra_data" method.